packages/mcp-server/src/server.ts

```diff
@@ -270,6 +276,9 @@ export async function configureServer({
                 name: `tools/call ${tool.name}`,
                 attributes: {
                   "mcp.tool.name": tool.name,
+                  ...(context.userAgent && {
```

Comment:
we'll remove this once sentry's core instrumentation does this at the mcp.server level